osu.Game.Rulesets.Mania/Configuration/ManiaRulesetConfigManager.cs

```diff
@@ -15,32 +15,51 @@ public class ManiaRulesetConfigManager : RulesetConfigManager<ManiaRulesetSettin
         public ManiaRulesetConfigManager(SettingsStore? settings, RulesetInfo ruleset, int? variant = null)
             : base(settings, ruleset, variant)
         {
+            migrate();
         }
 
         protected override void InitialiseDefaults()
         {
             base.InitialiseDefaults();
 
-            SetDefault(ManiaRulesetSetting.ScrollTime, 1500.0, DrawableManiaRuleset.MIN_TIME_RANGE, DrawableManiaRuleset.MAX_TIME_RANGE, 5);
+#pragma warning disable CS0618
+            // Although obsolete, this is still required to populate the bindable from the database in case migration is required.
+            SetDefault<double?>(ManiaRulesetSetting.ScrollTime, null);
+#pragma warning restore CS0618
+
+            SetDefault(ManiaRulesetSetting.ScrollSpeed, 8, 1, 40);
             SetDefault(ManiaRulesetSetting.ScrollDirection, ManiaScrollingDirection.Down);
             SetDefault(ManiaRulesetSetting.TimingBasedNoteColouring, false);
         }
 
+#pragma warning disable CS0618
+        private void migrate()
```

Comment:
I'd either combine this method into `InitialiseDefaults`, or at very least move the `migrate()` call into `InitialiseDefaults`.

Having it in the `ctor` when `Load` and `InitialiseDefaults` are called from a base constructor makes me very uneasy.

Reply:
I did it this way to follow an existing implementation... https://github.com/ppy/osu/blob/master/osu.Game/Configuration/OsuConfigManager.cs